modules/extension/xsd/xsd-core/src/main/java/org/geotools/xml/Parser.java

```diff
@@ -50,6 +50,8 @@
 import org.xml.sax.SAXException;
 import org.xml.sax.helpers.NamespaceSupport;
 
+import com.sun.org.apache.xerces.internal.impl.Constants;
```

Comment:
Wondering out loud here, will this be an issue with Java 9? However I could not find a replacement that is not in com.sun...